packages/neo-one-client/src/user/LocalUserAccountProvider.ts

```diff
@@ -93,8 +93,8 @@ export interface KeyStore {
   readonly accounts$: Observable<ReadonlyArray<UserAccount>>;
   readonly getAccounts: () => ReadonlyArray<UserAccount>;
   readonly selectAccount: (id?: UserAccountID, monitor?: Monitor) => Promise<void>;
-  readonly deleteAccount: (id: UserAccountID, monitor?: Monitor) => Promise<void>;
-  readonly updateAccountName: (options: UpdateAccountNameOptions) => Promise<void>;
+  readonly deleteAccount?: (id: UserAccountID, monitor?: Monitor) => Promise<void>;
```

Comment:
These should still be non-optional. Revert these changes. Your implementation should just throw an error if they're called. Whether or not the client can actually these methods is controlled by the `configurableName` and `deletable` properties on `UserAccount` which they must check before using these methods (otherwise, they can expect an error to be thrown)